pkg/query/endpointset.go

```diff
@@ -146,6 +147,7 @@ func (es *grpcEndpointSpec) fillExpectedAPIs(componentType component.Component,
 				MinTime: mintime,
 				MaxTime: maxTime,
 			},
+			Query: &infopb.QueryInfo{},
```

Comment:
Note that without `--endpoint` it means that we assume that all Stores implement QueryAPI :s